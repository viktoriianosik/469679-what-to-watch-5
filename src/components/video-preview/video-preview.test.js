import React from "react";
import renderer from "react-test-renderer";
import VideoPreview from "./video-preview";

it(`Render component VideoPreview correctly`, () => {
  const tree = renderer.create(
      <VideoPreview
        isPlaying={true}
        muted={true}
        src={`img/*`}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
