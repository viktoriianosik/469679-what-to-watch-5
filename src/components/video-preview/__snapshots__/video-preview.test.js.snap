// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component VideoPreview correctly 1`] = `
<video
  autoPlay={true}
  className="player__video"
  muted={true}
  src="img/*"
/>
`;
